docs/SearchSpaceSpec.md

```diff
@@ -39,13 +39,15 @@ The candidate type and value for variable is here:
    * Suitable for a discrete value with respect to which the objective is still somewhat "smooth", but which should be bounded both above and below. If you want to uniformly choose integer from a range [low, high], you can write `_value` like this: `[low, high, 1]`.
 <br/>
 
-* {"_type":"loguniform","_value":[low, high]}
-   * Which means the variable value is a value drawn according to exp(uniform(low, high)) so that the logarithm of the return value is uniformly distributed.
-   * When optimizing, this variable is constrained to the interval [exp(low), exp(high)].
+* {"_type":"loguniform","_value":[low, high, base]}
```

Comment:
I just wonder, do we need to change the search_space.json new design? because I can't tell did I need to add "base" value in the future?

Reply:
there is default value here, so no bad to support it, right?